src/padplus-manager/padplus-manager.ts

```diff
@@ -823,6 +825,90 @@ export class PadplusManager extends EventEmitter {
   /**
    * Contact Section
    */
+
+  public async newTag (tag: string): Promise<string> {
+    if (!this.padplusContact) {
+      throw new Error(`no padplusContact`)
+    }
+
+    return this.padplusContact.newTag(tag)
+  }
+
+  public async addTag (tagId: string, contactId: string): Promise<void> {
+    if (!this.padplusContact) {
+      throw new Error(`no padplusContact`)
+    }
+    const tags = await this.tags(contactId)
+    const tagsId = tags.map(tag => tag.id)
+    const allTagsId = tagsId.length === 0 ? tagId : tagsId.join(',') + ',' + tagId
+    await this.padplusContact.addTag(allTagsId, contactId)
+  }
+
+  public async tags (contactId: string): Promise<TagPayload []> {
+    if (!this.cacheManager) {
+      throw new Error(`no cacheManager`)
+    }
+
+    const contact = await this.cacheManager.getContact(contactId)
+    if (!contact) {
+      throw new Error(`can not get contact by this contactId: ${contactId}`)
+    }
+    const labelsId = contact.labelLists
+    const labelIdsList = labelsId.split(',')
+
+    const allLabel: TagPayload[] = await this.tagList()
```

Comment:
If we want to use `tag` in wechaty, we should keep the variables are `tag`, not `label` to avoid unnecessary confusing.